src/incremental/serialize.ml

```diff
@@ -1,10 +1,7 @@
 open Prelude
 
 (* TODO: GoblintDir *)
-let version_map_filename = "version.data"
-let cil_file_name = "ast.data"
-let solver_data_file_name = "solver.data"
-let analysis_data_file_name = "analysis.data"
+let incremental_data_file_name = "analysis.data"
 let results_dir = "results"
 let results_tmp_dir = "results_tmp"
```

Comment:
Is the whole business with two directories and renaming them still necessary?

Reply:
No, actually not.